src/image_builder.py

```diff
@@ -138,7 +168,7 @@ def image_builder(buildspec):
                 }
             }
         )
-
+        
```

Comment:
nit: Please remove trailing whitespace. This is possibly an IDE configuration that can be used to automatically delete trailing whitespace in a file when it is saved.